modules/deb.sh

```diff
@@ -130,8 +130,8 @@ function deb_install_package_in_chroot() {
     run_in_chroot_and_log_cmd_noexit "$dest_dir" "env \
         DEBIAN_FRONTEND=noninteractive \
         DEVICE_TYPE=${MENDER_DEVICE_TYPE} \
-        dpkg --install /tmp/$(basename $deb_package)" || ret=$?
-    rm -f "$dest_dir/tmp/$(basename $deb_package)"
+        dpkg --install '/tmp/$(basename "$deb_package")'" || ret=$?
```

Comment:
You should use double quotes.

With single quotes, the `$(...` and `$deb...` are not expanded and just passed verbatim. This works only because at the end, in `run_and_log_cmd_noexit`, we have an `eval` command which will then expand.

Reply:
IMO this is a good fix and we should mark the commit as `fix`